requirements.txt

```diff
@@ -1,2 +1,2 @@
-pulpcore>=3.40.1,<3.55
+pulpcore>=3.45.1,<3.55
```

Comment:
Bumping minimum pulpcore version to 3.45.1 because our model depends on `0116_alter_remoteartifact_md5_alter_remoteartifact_sha1_and_more`. Without it we get the error:
```
    django.db.migrations.exceptions.NodeNotFoundError: Migration ostree.0008_add_domain_support dependencies reference nonexistent parent node ('core', '0116_alter_remoteartifact_md5_alter_remoteartifact_sha1_and_more')
```